qualisys_driver/CMakeLists.txt

```diff
@@ -24,6 +22,19 @@ find_package(mocap4r2_msgs REQUIRED)
 find_package(geometry_msgs REQUIRED)
 
 
+include(FetchContent)
+if (NOT TARGET qualisys_cpp_sdk)
+  message(STATUS "${PROJECT_NAME}: `qualisys_cpp_sdk` targets not found. Attempting to fetch contents...")
+  FetchContent_Declare(
+    qualisys_cpp_sdk
+    GIT_REPOSITORY https://github.com/incebellipipo/qualisys_cpp_sdk
```

Comment:
This should ideally be a fork and it should be something like: mocap4ros2/qualisys_cpp_sdk 
@fmrico